internal/rollup/rollup_bundle.bzl

```diff
@@ -150,15 +208,11 @@ ROLLUP_ATTRS = {
         single_file = True),
 }
 
-ROLLUP_OUTPUTS = {
```

Comment:
We can't land it if we remove these. Can we still output the files for the "main" bundle or something?

Reply:
We could make it so that the rule could output both a regular bundle or a code split one but without a devserver that can support code splitting is there any value in a landing the production bundler at this point?

Reply:
I think we need to land this somewhere with CI so that it doesn't rust, and people can try it and build on it.
I think there are some users who would be satisfied with lazy-loading only for production, if they have simple enough code splitting and are okay with having to debug issues in production mode.
I dunno it seems like  a tough call.